### PR TITLE
Reference cards

### PR DESCRIPTION
**Pull request by jmthompson2015**
Added missing reference card images. Used TinyPNG on images.

### CODE REVIEW

File: data/reference-cards.js [hunk 1]
```diff
@@ -1,160 +1,176 @@
...
+      "id": 13,
+      "title": "Bomb Tokens",
+      "subtitle": "Reference Card #03",
+      "image": "reference-cards/BombTokens-03.png"
+  },
+   {
+      "id": 14,
+      "title": "Tractor Beam Token",
+      "subtitle": "Reference Card",
+      "image": "reference-cards/TractorBeanToken.png"
```
Comment (guidokessels) [inline data/reference-cards.js]: This needs to be changed to `reference-cards/TractorBeamToken.png` as you renamed the image.

Comment (guidokessels) [reply data/reference-cards.js]: This is why the build is failing.

**Comment by guidokessels** [comment]
Thanks for this! There's a small error that needs to be fixed to make the build pass. I added a comment to the line that needs to be changed.

Could you also check the whitespace? Right now the git diff shows that every line has changed because the file's indenting has changed.

**Comment by jmthompson2015** [comment]
I fixed the filename and the whitespace issue (turned atom-beautify off). And added a couple more reference cards while I was at it. Please take another look.

**Comment by guidokessels** [comment]
Thanks a lot!

**Pull request merged**